tiatoolbox/utils/env_detection.py

```diff
@@ -365,7 +362,7 @@ def pixman_versions() -> tuple[list, str]:
             determined.
 
     """
-    versions = []
+    versions: list[Any] = []
```

Comment:
shouldn't this be list[int] ? as it says in docstring?

Reply:
Yeah it should be list[int]